larq_compute_engine/core/bgemm_kernels_common.h

```diff
@@ -53,13 +53,13 @@ struct BinaryKernelParams {
   std::int32_t clamp_max;
   std::int32_t backtransform_add;
   std::uint8_t flags;
-  T dst_tmp_buf[LhsCols * RhsCols];
+  float dst_tmp_buf[LhsCols * RhsCols];
```

Comment:
This I think is a mistake on master, as in this case T is the bitpacked type not the destination type.